components/automate-ui/src/app/modules/token/token-list/api-token-list.component.html

```diff
@@ -47,7 +47,7 @@
               <chef-table-header-cell *ngIf="isIAMv2$ | async">Projects</chef-table-header-cell>
               <chef-table-header-cell>Created Date</chef-table-header-cell>
               <chef-table-header-cell>Status</chef-table-header-cell>
-              <chef-table-header-cell class="controls"></chef-table-header-cell>
+              <chef-table-header-cell class="three-dot-column"></chef-table-header-cell>
```

Comment:
PR Bonus: A recent PR changed all the other resources to "three-dot-column". This one was likely added right about the same time, using the old name, so updated this here.